src/components/Notes/Search.jsx

```diff
@@ -0,0 +1,14 @@
+import { MdSearch } from "react-icons/md";                                              
+const Search =({handleSearchNote})=>{
```

Comment:
We prefer to call the callbacks with `on`.
```suggestion
const Search =({ onSearch })=>{
```

We use prefix `handle` to define the handler itself as you did in Notes.jsx